graphql-server/pkg/middlewares/auth.go

```diff
@@ -0,0 +1,49 @@
+package middlewares
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/gin-gonic/gin"
+	"github.com/oursky/likedao/pkg/config"
+	pkgContext "github.com/oursky/likedao/pkg/context"
+	"github.com/oursky/likedao/pkg/handlers"
+	"github.com/oursky/likedao/pkg/models"
+)
+
+func Authentication(
+	config config.Config,
+) gin.HandlerFunc {
+	return func(c *gin.Context) {
+
+		if gin.Mode() == gin.DebugMode {
+			dummyAddress := []byte{0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 1, 0}
```

Comment:
Use header to manually override signed in user address?
with this we can debug for unauthed case?